index.html

```diff
@@ -30,19 +30,127 @@
         </nav>
     </header>
     <section>
-        <div class="container">
-            <h1 class="intro">I'm Hammas Glad to see you!</h1>
-            <p class="description">I’m a software developer! I can help you build a product , feature or website
-                Look through some of my work and experience! If you like what you see and have a project you need coded,
-                don’t hestiate to contact me. </p>
-            <div class="container2">
-                <p class="connect">LET'S CONNECT</p>
-                <ul class="icon-list">
-                    <li><a href="#" class="fa fa-twitter fa-color" aria-hidden="true"></a></li>
-                    <li><a href="#" class="fa fa-linkedin fa-color" aria-hidden="true"></a></li>
-                    <li><a href="#" class="fa fa-medium fa-color" aria-hidden="true"></a></li>
-                    <li><a href="#" class="fa fa-github fa-color" aria-hidden="true"></a></li>
-                    <li> <a href="#" class="fa fa-angellist fa-color" aria-hidden="true"></a></li>
+            <div class="container homepage">
+                <h1 class="intro">I'm Hammas Glad to see you!</h1>
+                <p class="description">I’m a software developer! I can help you build a product , feature or website
+                    Look through some of my work and experience! If you like what you see and have a project you need
+                    coded,
+                    don’t hestiate to contact me. </p>
+                <div class="container2">
+                    <p class="connect">LET'S CONNECT</p>
+                    <ul class="icon-list">
+                        <li><a href="#" class="fa fa-twitter fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-linkedin fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-medium fa-color" aria-hidden="true"></a></li>
+                        <li><a href="#" class="fa fa-github fa-color" aria-hidden="true"></a></li>
+                        <li> <a href="#" class="fa fa-angellist fa-color" aria-hidden="true"></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="cr-grid">
```

Comment:
- [ ] Let's ensure we follow good practice by using more semantic tags  (`section`  tag) for this container.